src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -603,20 +595,20 @@ default String checkPromptEquals(String expectedPromptText, double waitFor, doub
      * @param timeTook              - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: if the prompt is present, the text of the prompt
      */
-    default String checkPromptMatches(String expectedPromptPattern, double waitFor, double timeTook) {
+    String checkPromptMatches(String expectedPromptPattern, double waitFor, double timeTook) {
         // check for the object to be present
         String prompt;
-        boolean isPromptPresent = getApp().is().promptPresent();
+        boolean isPromptPresent = this.app.is().promptPresent();
         if (isPromptPresent) {
-            prompt = getApp().get().prompt();
+            prompt = this.app.get().prompt();
         } else {
-            getReporter().fail("to find prompt with the text matching pattern <b>" + expectedPromptPattern + ON_PAGE, waitFor, NO_PROMPT, timeTook);
+            this.reporter.fail("to find prompt with the text matching pattern <b>" + expectedPromptPattern + ON_PAGE, waitFor, NO_PROMPT, timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "to find prompt with the text matching pattern <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)